CodegenCLI/.gitignore

```diff
@@ -0,0 +1,9 @@
+.DS_Store
```

Comment:
Does the CLI directory need it's own gitignore file? I thought gitignore only worked in the repo root?

Reply:
You're correct, it doesn't. I just did a quick test and it's actually created by the `swift package init` command. My mistake for not being more aware of it getting committed - I'll get rid of it.